.travis.yml

```diff
@@ -0,0 +1,5 @@
+language: node_js
+node_js:
+  - "7.10.1"
+script:
+  - npm run test
```

Comment:
you might also want to use the cache option so that it doesnt reinstall node_modules all the time. this reduces build times tremendously

https://docs.travis-ci.com/user/caching/